src/Types.ts

```diff
@@ -60,6 +60,8 @@ export interface QRDrawingConfig {
     logoMargin: number;
     logoCornerRadius: number;
     maskedDots: boolean;
+    designStyle?:string;
```

Comment:
@hritikaggarwal31 Are we still using these attributes?

Reply:
No, removed these attributes.